Артём Швецов

shvetsovart opened the pull request:
Не надеюсь на закрытие PR, но на всякий случай - это проект без хардкорной части. Не хочется переделывать всё сразу, включая хардкор, поэтому делаю PR сейчас. 

P.S. Я правильно понимаю, что баллы/респект за хардкор начисляются только при выполнении всех пунктов (скрытие текста, смена шрифта, смена размера, смена темы, скрытая шалость), или достаточно реализовать часть из этого?

https://codesandbox.io/p/devbox/d83l97

shvetsovart committed:
task2 completed (no hardcore)

Не надеюсь на закрытие PR, но на всякий случай - это проект без хардкорной части. Не хочется переделывать всё сразу, включая хардкор, поэтому делаю PR сейчас.

shvetsovart committed:
fixed codestyle

shvetsovart committed:
part of second page added

shvetsovart commented:
Только сейчас заметил, что нужно было сверстать всю газету, а не только первую страницу. Можно, пожалуйста, дослать оставшуюся часть вместе с хардкором?

shvetsovart committed:
fixed codestyle

shvetsovart committed:
border to pages added

shvetsovart committed:
added height, width, deleted second page

shvetsovart commented:
Добавил высоту и ширину для фотографий. Насчёт span и ::first-letter: погуглил, поGPTшил и понял, что с помощью first-letter не получится сделать то, что Я хочу. Мне нужно, чтобы текст как бы обтекал эту букву в случае с Илоном Маском и буквой "D", а в случае с основным текстом Павла Дурова и буквой "R", чтобы весь текст был правее этой буквы. Кажется, с помощью ::first-letter такого не сделаешь, можно разве что изменить стиль первой буквы.

P.S. нашёл шрифт заголовка Daily Prophet - называется MuggleNews. Юзать его, конечно, не буду xD (хочется оставить эту красивую The...)

mokhov commented:
Мне кажется свёрстанный у тебя вариант не очень оптимально использует пространство под первой буквой. Сделал как бы я сверстал (и как обычно в газетах) и как у тебя 
https://dabblet.com/gist/cd80a9a125849548f3c9075660e8175e

shvetsovart commented:
Спасибо большое, это прекрасно!

вт, 24 сент. 2024 г., 18:45 Oleg Mokhov ***@***.***>:

> Мне кажется свёрстанный у тебя вариант не очень оптимально использует
> пространство под первой буквой. Сделал как бы я сверстал (и как обычно в
> газетах) и как у тебя
> https://dabblet.com/gist/cd80a9a125849548f3c9075660e8175e
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/itmo-2024/task2/pull/2#issuecomment-2371677032>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGIY53OIJK4NAVTDTDX42KDZYGCLHAVCNFSM6AAAAABOTRLR5OVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGNZRGY3TOMBTGI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

shvetsovart committed:
big fixes (still not hardcore)

Used knowledge from 3rd lecture, fixed title being separated into several words, new font styles, better looking, etc. Still not hardcore.

shvetsovart committed:
fixed main title issues

On low-width devices/windows the last line was devided in two which led to printing on two pages and other issues - fixed. Some codestyle fixes.

shvetsovart committed:
added hardcore

mokhov commented:
Принято
